src/window.jl

```diff
@@ -18,9 +18,19 @@ const window_defaults = @d(:url => "about:blank",
                            "use-content-size" => true,
                            :icon => Pkg.dir("Blink", "deps", "julia.png"))
 
+const windowhooks = Function[]
+
+windowinit(f) =
+    push!(windowhooks, f)
+
 function Window(a::AtomShell, opts::Associative = Dict())
   id = @js a createWindow($(merge(window_defaults, opts)))
-  return Window(id, a)
+  w = Window(id, a)
+  println("Exec hooks", windowhooks)
+  for f in windowhooks
+      f(w)
+  end
```

Comment:
Sure, I'd like to do that eventually, but it won't be a good option until windows have faster communication. Even so, it'd be nice to have the option to open completely untouched windows.